app/assets/stylesheets/global/_helper.scss

```diff
@@ -2,30 +2,53 @@
 // helper classes
 //--------------------------------------------------------------------------------
 // - backgrounds
+// - display
 // - text
 //--------------------------------------------------------------------------------
-
-
-//--------------------------------------------------------------------------------
 // backgrounds
 //--------------------------------------------------------------------------------
-.bg__grey {
```

Comment:
Again if we can split this out into smaller files in a helpers folder where each file does just one thing, that would improve the readability so much. As Shakespeare said; brevity is the soul of code.